# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/MockConduit.h
# Commit: b99b71e755e4
# Full Hash: b99b71e755e470a5dd152a5d6313f3002b2b9ba9
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-12-18 09:43:08
# Description:
#   Bug 1744081 - Move VideoConduit::Shutdown asyncness ownership into VideoConduit. r=bryce,ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D133933
# ==============================================================================

diff -r f69c775a1024 -r b99b71e755e4 media/webrtc/signaling/gtest/MockConduit.h
--- a/media/webrtc/signaling/gtest/MockConduit.h	Fri Dec 17 20:29:15 2021 +0000
+++ b/media/webrtc/signaling/gtest/MockConduit.h	Fri Dec 17 20:34:18 2021 +0000
@@ -48,7 +48,7 @@
   MOCK_METHOD2(SendSenderRtcp, bool(const uint8_t*, size_t));
   MOCK_METHOD2(SendReceiverRtcp, bool(const uint8_t*, size_t));
   MOCK_METHOD2(DeliverPacket, void(rtc::CopyOnWriteBuffer, PacketType));
-  MOCK_METHOD0(Shutdown, void());
+  MOCK_METHOD0(Shutdown, RefPtr<GenericPromise>());
   MOCK_METHOD0(AsAudioSessionConduit, Maybe<RefPtr<AudioSessionConduit>>());
   MOCK_METHOD0(AsVideoSessionConduit, Maybe<RefPtr<VideoSessionConduit>>());
   MOCK_CONST_METHOD0(GetCallStats, Maybe<webrtc::Call::Stats>());